extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -75,8 +75,13 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
 
     const theme = this.element.getAttribute('theme');
     if (theme && AttachmentTheme.DARK === theme.toLowerCase()) {
-      this.headerEl_.classList.add(DARK_THEME_CLASS);
-      this.element.classList.add(DARK_THEME_CLASS);
+      if (isPageAttachmentUiV2ExperimentOn(this.win)) {
+        this.headerEl_.setAttribute('theme', theme.toLowerCase());
```

Comment:
nit: you can change the `const theme = this.element.getAttribute('theme');` in line 76 to use optional chaining `const theme = this.element.getAttribute('theme')?.toLowerCase();` and get rid of all the `toLowerCase()` calls below.